javascript-practice/src/index.html

```diff
@@ -19,7 +19,40 @@
 		<!-- END HERO SECTION -->
 
 		<!-- START INTRO SECTION -->
-		<section></section>
+		<section class="container intro">
+			<h4 class="intro-title">What makes our brand different</h4>
+			<ul class="intro-list">
+				<li class="intro-item">
+					<img src="assets/icons/delivery.svg" alt="icon delivery" />
+					<h4 class="item-title">Next day as standard</h4>
```

Comment:
Why this one is h4? it below the h4, should check it

Reply:
I checked the design as h4 and I have the definition of h4 in PR of typography